.github/workflows/CD-adot-operator.yml

```diff
@@ -40,10 +42,32 @@ jobs:
 
       - name: Version checking
         id: version
-        run: VERSION="${{ github.event.inputs.version }}" bash tools/release/adot-operator-images-mirror/version-check.sh
+        run: |
+          VERSION="${{ github.event.inputs.version }}" bash tools/release/adot-operator-images-mirror/version-check.sh
+          echo "::set-output name=version::$VERSION"
 
       - name: Mirror ADOT operator
         if: ${{ steps.version.outputs.update-operator == 'true' }}
         run: cd tools/release/adot-operator-images-mirror && go run ./
         env:
           AWS_SDK_LOAD_CONFIG: true
+
+  push-to-private-ecr:
```

Comment:
We should probably rework the mirroring tool to push to private ECR first and then public then change this to reflect that

Reply:
Sure, I understood your point. 
Scenario  - if there is a decision to not ship the operator image  while the upstream `operator image` has vulnerability, in that case, that functionality needs to separated from CD.